### ISSUE TITLE
Makefile adjustments for packaging

### ISSUE THREAD

**Issue by lmartinez-mirror** [Dr-Noob/cpufetch#117]
tl;dr: see #116 

Hi, just wanted to suggest some changes for the Makefile to be more packager-friendly:

* The Makefile installs LICENSE to a hard-coded directory despite the prefixes. I suggest removing the install for it altogether and let packagers handle it themselves. I understand you did this because you maintain `cpufetch-git` on the AUR, but this should ideally be done in the PKGBUILD, not in the Makefile.
* LDFLAGS should be appended to compilation. Arch has guidelines for compiling and likes their binaries hardened with full RELRO; other packagers for other distros may have linker flags of their own that they may need to add.
* Maybe install the man pages without compression? At least on Arch, any packages that install man pages automatically gzip them.

**Comment by Dr-Noob** [comment]
Hi, thank you for your comments.

I've changed the Makefile so many times because packaging, and it is becoming really painful. Regarding your changes:
* Understandable, I will change it. When I read [how to create a package](https://wiki.archlinux.org/title/Creating_packages#package()) I thought that the best approach  was to include everything inside the `install` target of the Makefile, but now I agree that it is not since not there are packages for other distros, not only Arch.
* Understandable again but not very clear in the documentation. [Here](https://wiki.archlinux.org/title/Arch_package_guidelines/Security) talks about LDFLAGS, but does not explicitly say that one should allow passing LDFLAGS. Maybe this is explained on another page, but I didn't find it. I will change it too.
* Do you mean to allow non-compressed man pages in other package management systems? Does it make sense? If I should change the entire `install` target in the Makefile maybe it's a better idea to leave it as it is for Arch and expect other package managers to do their own install process instead of using `make install`.
